community/opendkim/opendkim.initd

```diff
@@ -1,16 +1,17 @@
 #!/sbin/openrc-run
 
+owner=opendkim
 pidfile=/run/opendkim/opendkim.pid
-
+cfgfile=/etc/opendkim/opendkim.conf
 command=/usr/sbin/opendkim
-command_args="-f ${EXTRA_OPTS}"
-command_background=yes
+command_args="$command_args -u $owner -P $pidfile"
```

Comment:
Why have you changed it? The current code does not work? It’s generally better and conceptually cleaner to let the daemonization job and handling of pidfile to rc system. It’s just a relict of shitty SysVinit that many programs automatically daemonizes itself and creates pidfiles (actually the whole concept of pidfile is just a legacy).